### PR TITLE
Responsive-Changes

### PR DESCRIPTION
**Pull request by mayank-del**
## Issue that this pull request solves

 Closes: #721 

## Proposed changes

to make home page responsive

### Brief description of what is fixed or changed

I changed Index.html page, OverView.css and overview.jsx 

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the cha
Tch-comm
nges made in case of change in user interface

## Other information

To contribute as a member of tech community

### CODE REVIEW

File: frontend/src/pages/Home/components/Overview/overview.module.scss [hunk 1]
```diff
@@ -63,30 +63,50 @@
     object-fit: cover;
   }
 }
+/* @media (max-width:768px){
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Home/components/Overview/overview.module.scss:66]: Can we please remove these commented lines?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@mayank-del as you started on responsiveness. Can you please fix the `About Us` page responsiveness as well?

**Comment by mayank-del** [comment]
Yaa sure sure

**Comment by mayank-del** [comment]
@Kajol-Kumari  please check it once, I think everything is fine now.

File: frontend/src/pages/Home/components/Overview/overview.module.scss [hunk 2]
```diff
@@ -64,29 +64,45 @@
...
 
   .overview img {
+    order: -1;
     min-width: auto;
     width: 180px;
-    margin-left: 11px;
+  }
+
+  .overview h1 {
+    text-align: center;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Home/components/Overview/overview.module.scss:88]: this shouldn't be centered aligned we want it just above the dashed line[i.e same as it was before]

File: frontend/src/pages/About/about.module.scss [hunk 3]
```diff
@@ -97,14 +97,17 @@
 .row1 > * {
   flex: 49% 49%;
 }
-
+h1 {
+  text-align: center;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/About/about.module.scss:101]: same comment as below, we don't want it centred aligned. Please make it same as it was before

Comment (Kajol-Kumari) [inline frontend/src/pages/About/about.module.scss:101]: same comment as below, we don't want it centred aligned. Please make it same as it was before

File: frontend/src/pages/About/About.jsx [hunk 4]
```diff
@@ -40,7 +40,7 @@ export const About = (props) => {
       >
         <div className="row">
           <div className="col-lg-5">
-            <h1>Who are we and what we do?</h1>
+            <h1 className="first">Who are we and what we do?</h1>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/About/About.jsx:43]: please make it stick to the left similar to [this](https://github.com/HITK-TECH-Community/Community-Website/pull/744/files#diff-9629027d328ffb062e91696c8962c46a51ad143e1f6224b2473a6afd51c540f3R52) line and same goes for the heading [resent in the home page

Comment (Kajol-Kumari) [inline frontend/src/pages/About/About.jsx:43]: please make it stick to the left similar to [this](https://github.com/HITK-TECH-Community/Community-Website/pull/744/files#diff-9629027d328ffb062e91696c8962c46a51ad143e1f6224b2473a6afd51c540f3R52) line and same goes for the heading [resent in the home page

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@mayank-del please do the requested changes. Also please format the css file with each style object separated by one blank line.

File: frontend/src/pages/Home/components/Overview/overview.module.scss (hunk 2 above)
Comment (mayank-del) [reply frontend/src/pages/Home/components/Overview/overview.module.scss:88]: But it was   aligned to center from  begining , kindly check it at your repo

File: frontend/src/pages/About/about.module.scss (hunk 3 above)
Comment (mayank-del) [reply frontend/src/pages/About/about.module.scss:101]: This will be done

File: frontend/src/pages/About/About.jsx (hunk 4 above)
Comment (mayank-del) [reply frontend/src/pages/About/About.jsx:43]: I am not understanding, what changes you are requesting here.

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for making the requested changes @mayank-del
LGTM 👍

**Pull request merged**